vms/proposervm/service.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package proposervm
+
+import (
+	"fmt"
+	"net/http"
+
+	"go.uber.org/zap"
+
+	"github.com/ava-labs/avalanchego/api"
+
+	avajson "github.com/ava-labs/avalanchego/utils/json"
+)
+
+type ProposerAPI struct {
+	vm *VM
+}
```

Comment:
nit: naming is odd here - this is `proposervm.ProposerAPI`, but the file-name is `service` (which is idiomatic w/ rpc naming). Should this be `proposervm.service` instead? Additionally it looks like we don't need to export this type the way it's currently being used.